mmv1/third_party/terraform/services/gkehub2/resource_gke_hub_scope_rbac_role_binding_test.go

```diff
@@ -59,7 +61,6 @@ resource "google_gke_hub_scope_rbac_role_binding" "scoperbacrolebinding" {
   labels = {
       key = "value" 
   }
-  depends_on = [google_gke_hub_scope.scoperbacrolebinding]
```

Comment:
With this removed, you'll need to establish the implicit dependency by updating the scope_id above (ie. to `google_gke_hub_scope.scoperbacrolebinding.scope_id`)